app/models/listing.py

```diff
@@ -39,6 +44,7 @@ def remove_from_carts(self):
             db.session.delete(cart_item)
         db.session.commit()
 
+
```

Comment:
pep8 (i think too many enters between methods...idk if you want to fix or not, dont really care.